serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -67,39 +49,20 @@ RUN mkdir -p /opt/djl/conf && \
 COPY config.properties /opt/djl/conf/config.properties
 COPY partition /opt/djl/partition
 
-
-COPY distribution[s]/ ./
-RUN mv *.deb djl-serving_all.deb || true
-
 # Install OpenMPI and other deps
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get install -y g++ wget unzip openmpi-bin libopenmpi-dev libffi-dev git-lfs rapidjson-dev graphviz && \
+RUN apt-get update && apt-get upgrade -y && apt-get install -y g++ wget unzip openmpi-bin libopenmpi-dev libffi-dev git-lfs rapidjson-dev graphviz cuda-compat-12-8 && \
     scripts/install_python.sh ${python_version} && \
     pip3 cache purge && \
     apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
-# Install PyTorch
-# Qwen needs transformers_stream_generator, tiktoken and einops
-RUN pip install torch==${TORCH_VERSION} transformers==${transformers_version} accelerate==${accelerate_version} peft==${peft_version} sentencepiece \
-    mpi4py cuda-python==${cuda_python_version} onnx polygraphy pynvml==${pynvml_verison} datasets==${datasets_version} pydantic==${pydantic_version} scipy torchprofile bitsandbytes ninja \
-    transformers_stream_generator einops tiktoken jinja2 graphviz blobfile colored h5py strenum pulp flax easydict && \
-    pip3 cache purge
-
-# Install TensorRT and TRT-LLM Deps
-RUN pip install --no-cache-dir --extra-index-url https://pypi.nvidia.com tensorrt==${tensorrtlibs_version} janus==${janus_version} nvidia-modelopt==${modelopt_version} && \
-    pip install --no-deps ${trtllm_wheel} && \
-    pyver=$(echo $python_version | awk -F. '{print $1$2}') && \
-    pip3 cache purge
+# We install TRTLLM separately because it is hosted in a different pypi index
+COPY trtllm-container-requirements.txt requirements.txt
+RUN pip install tensorrt_llm==${trtllm_version} --extra-index-url https://pypi.nvidia.com && \
```

Comment:
`trtllm_version` might be different from the one in `requirements.txt`, If we have to hardcode in `requirements.txt`, why not move it completely to `requirements.txt` file?

Why we need to keep this file in `/opt/djl` folder?

Reply:
I think for the trtllm container, the set of requirements is so small I'll just install them in-line in the docker file. 

The main reason for separation is that --extra-index-url applies globally to all requirements, and we have no control over where a requirement is sourced from and it could be tampered with if the index is compromised (appsec).

will change this to:
```
RUN pip install tensorrt_llm==${trtllm_version} --index-url https://pypi.nvidia.com && \
    pip install tensorrt_llm==${trtllm_version} uvloop ninja
```